<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IonSourceWidget</class>
 <widget class="QWidget" name="IonSourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>GC</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Housing Temp (C)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>GC I/F Temp (C)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSlider" name="horizontalSlider_01">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QSlider" name="horizontalSlider_02">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_01"/>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_02"/>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label01">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLabel" name="label02">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Trap (mA)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Trap (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label11">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="horizontalSlider11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Filament (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Filament (mA)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ionization Voltage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox11"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ion Source</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Chamber (uA)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox12"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox13"/>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox14"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox15"/>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox16"/>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="horizontalSlider12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="horizontalSlider13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSlider" name="horizontalSlider14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSlider" name="horizontalSlider15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSlider" name="horizontalSlider16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label12">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label13">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label14">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label15">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label16">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="3">
      <widget class="QLabel" name="label22">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label21">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox21"/>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="horizontalSlider21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pullout (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Ion Transfer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Lens 1 (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Lens 2 (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label23">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox22"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox23"/>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="horizontalSlider22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="horizontalSlider23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Lens 4 (V)l</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Lens 3 (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Lens 5 (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Pre-filter DC (V)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label24">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label25">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label26">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label27">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSlider" name="horizontalSlider24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSlider" name="horizontalSlider25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSlider" name="horizontalSlider26">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QSlider" name="horizontalSlider27">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox24"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox25"/>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox26"/>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox27"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
